variables.tf

```diff
@@ -32,3 +32,9 @@ variable "tags" {
   default     = {}
   description = "Additional tags (e.g. `map('BusinessUnit`,`XYZ`)"
```

Comment:
please also update the quotes
Additional tags (e.g. `map('BusinessUnit','XYZ')`)